### PR TITLE
#2475 History and Bookmark fragments are not showing their title

### LINKED ISSUE #2475: History and Bookmark are not showing the correct title
**Describe the bug**
On History and Bookmark the title is Kiwix

**Expected behavior**
The screens should be appropriately titled

### PR DESCRIPTION
**Pull request by gouri-panda**
Fixes #2475

### CODE REVIEW

File: core/src/main/java/org/kiwix/kiwixmobile/core/page/PageFragment.kt [hunk 1]
```diff
@@ -123,7 +123,7 @@ abstract class PageFragment : OnItemClickListener, BaseFragment(), FragmentActiv
     activity.setSupportActionBar(toolbar)
     activity.supportActionBar?.apply {
       setDisplayHomeAsUpEnabled(true)
-      title = title
+      title = this@PageFragment.title
```
Comment (macgills) [inline core/src/main/java/org/kiwix/kiwixmobile/core/page/PageFragment.kt:126]: make sure to ignore the warning that is calling this `this` unnecessary or some unsuspecting good samaritan could easily reintroduce this bug

Comment (gouri-panda) [reply core/src/main/java/org/kiwix/kiwixmobile/core/page/PageFragment.kt:126]: @macgills I don't know why is this calling `this` is unnecessary? Do you want me to suppress this warning on deket?

Comment (gouri-panda) [reply core/src/main/java/org/kiwix/kiwixmobile/core/page/PageFragment.kt:126]: option 2 if we remove the title inside of the toolbar scope. which one is better?

Comment (macgills) [reply core/src/main/java/org/kiwix/kiwixmobile/core/page/PageFragment.kt:126]: the warning is very useful so we do not want to disable it, also it is not from detekt, it is an inspection. press alt+enter on it and you will get suppression options, you can suppress this statement only.

This is an unfortunate false positive, alternatively we could rename title and call it `screenTitle` or the like

**Review by macgills — APPROVED** [review]

**Pull request merged**